pcb/rules/F5_3.py

```diff
@@ -174,9 +224,17 @@ def check(self):
             for bad in self.bad_grid:
                 self.errorExtra(graphItemString(bad, layer=True, width=False))
 
+        # Check that courtyard is closed
+        if len(self.unconnected) > 0:
+            self.error("Courtyard should be closed.")
```

Comment:
Wrong wording. Use: Courtyard **must** be closed. (emphasis to make it clear what word to change.)